pyproject.toml

```
@@ -1,6 +1,7 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
+    "packaging==20.5",
```

I wonder if we should signal that this is needed for M1, what do you think? I think `arm64` is the correct one, `aarch64` is for linux.
```suggestion
    "packaging==20.5; platform_machine=='arm64'",
```

---

Makes sense.
Do you know what might be going on with "Build_Test/full" failure.